### PR TITLE
Feature: stat regression fix

### LINKED ISSUE #2853: Stat regression for stats using the stat content field
# Problem
It looks like there was a stat regression possibly introduced in https://github.com/uiowa/uiowa/pull/2734.  

Screen Shot 2021-01-12 at 2 23 06 PM

# Steps to reproduce
Check out stats on https://admissions.prod.drupal.uiowa.edu/.   The bug only exists if the stat has `stat__content`.  

# Proposed solution
- Update `uids_base` stat component 
- Integrate UIDS release from https://github.com/uiowa/uids/pull/458

### PR DESCRIPTION
**Pull request by bspeare**
This pull request resolves https://github.com/uiowa/uiowa/issues/2853.   This pr also includes a fix in `uids_base.libraries.yml` to adjust the weight of the `background.css` file to get this recent UIDS issue working https://github.com/uiowa/uids/issues/456 - you can view this update at http://sandbox.local.drupal.uiowa.edu/cards.  

# How to test

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
blt ds --site=admissions.uiowa.edu
```
1. Check that you have `3.1.8` of UIDS in your node_modules folder, I was having trouble locally getting this to `yarn install`. 
3. Test that stats on the home page are centered and that the hover/no-hover layout builder style is working. 
4. Test that jumpiness on stat grid on https://admissions.prod.drupal.uiowa.edu/ is fixed.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -981,9 +981,23 @@
...
+    gulp "^4.0.0"
+    gulp-imagemin "^7.1.0"
+    gulp-postcss "^8.0.0"
+    gulp-sass "^4.1.0"
+    gulp-sass-glob "^1.0.9"
+    gulp-sourcemaps "^2.6.4"
+
+"@uiowa/uids@uiowa/uids#3.1.8":
+  version "3.1.8"
+  resolved "https://codeload.github.com/uiowa/uids/tar.gz/1325323bd4634f3646472976be26196fd0a33460"
```
Comment (pyrello) [inline yarn.lock]: Is this a quirk of a new version of yarn? It seem like we shouldn't have multiple definitions for UIDS in here.

Comment (pyrello) [reply yarn.lock]: I removed the extra entry and it didn't seem to cause any issues.

**Review by pyrello — APPROVED** [review]
Tested per instructions and everything looks good.

**Comment by bspeare** [comment]
@pyrello Thanks! That `yarn.lock` issue has happened a few times now.  I'll add it to my list of yarn troubleshooting techniques.

**Pull request merged**